templates/footer.twig

```diff
@@ -1,74 +1,80 @@
-<footer class="site-footer" id="footer">
-	<div class="container">
-		<ul class="footer-social-media list-unstyled col-md-8 col-lg-5">
-			{% if social_overrides is defined and social_overrides|length %}
-				{% for social in social_overrides %}
-					<li>
-						<a href="{{ social.url }}"
-						   data-ga-category="Footer Navigation"
-						   data-ga-action="Social Icons"
-						   data-ga-label="{{ social.icon }}">
-							{{ social.icon|svgicon }}
-							<span class="visually-hidden">{{ social.icon }}</span>
-						</a>
-					</li>
-				{% endfor %}
-			{% else %}
-				{% for social in footer_social_menu %}
-					{% set name = "" %}
-					{% for class_name in social.classes %}
-						{% if 'fa-' in class_name %}
-							{% set name = class_name[3:] %}
-						{% else %}
-							{% set name = class_name %}
-						{% endif %}
-					{% endfor %}
-					<li>
-						{% set social_url = social.url %}
-						<a href="{{ social_url }}"
-						   data-ga-category="Footer Navigation"
-						   data-ga-action="Social Icons"
-						   data-ga-label="{{ social.title }}">
-							{{ name|svgicon }}
-							<span class="visually-hidden">{{ social.title }}</span>
-						</a>
-					</li>
-				{% endfor %}
-			{% endif %}
-		</ul>
+{% set follow_us = __( 'Follow us', 'planet4-master-theme' ) %}
```

Comment:
What is the reason for assigning this to a variable? The `__` function is used quite generally, I don't see a reason why this instance should be assigned.

Reply:
I agree